cotk/models/ngram_language_model.py

```diff
@@ -203,9 +214,14 @@ def perplexity(self, corpus):
 		Returns:
 		    (float): Perplexity when generating ``corpus``
 		'''
-		length = 0
-		entropy = 0
-		for sent in corpus:
-			entropy -= self.sent_log_prob(sent)
-			length += len(sent) + 1
-		return np.exp(entropy / length)
+		log_probs = []
+		if len(corpus) > 100:
```

Comment:
Refer to #295 ,  progress bar and multiprocess should not bind together.  If cpu_cout > 0 and corpus >100 use multiprocess;  corpus >100 use tqdm, regardless of cpu_count.